From 1ddc8a3dca2881bef56855bb7741e8814c3753f6 Mon Sep 17 00:00:00 2001
From: Adrian Reber <areber@redhat.com>
Date: Fri, 3 Dec 2021 16:59:56 +0000
Subject: [PATCH 071/120] test: another try to correctly fix the kernel version

We try to disable time namespace based testing for kernels older than
5.11. But we fail to come up with the correct if condition.

This changes (major <= 5) to (major < 5). There are no kernels with
major > 5 so currently the time namespace based are never run. This
should finally change it to run time namespace based tests on kernel
versions newer than 5.10.

Signed-off-by: Adrian Reber <areber@redhat.com>
---
 test/zdtm_ct.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/test/zdtm_ct.c b/test/zdtm_ct.c
index e8d45a9e7..e4b17a512 100644
--- a/test/zdtm_ct.c
+++ b/test/zdtm_ct.c
@@ -61,7 +61,7 @@ static int create_timens()
 	if (sscanf(buf.release, "%u.%u", &major, &minor) != 2)
 		return -1;
 
-	if ((major <= 5) || (major == 5 && minor < 11)) {
+	if ((major < 5) || (major == 5 && minor < 11)) {
 		fprintf(stderr, "timens isn't supported on %s\n", buf.release);
 		return 0;
 	}
-- 
2.34.1

